specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
```

Comment:
>paths [](start = 3, length = 5)

this version is dropping the following api.. The clients using that won't be able to move to new version of the SDK. Hope this is thought through. 

"/subscriptions/{subscriptionId}/providers/Microsoft.Marketplace/privateStoreClient/isPrivateClient": {

Reply:
Yes. it was thought through.